### ISSUE TITLE
Deinstallation not working properly

### ISSUE THREAD

**Issue by lisandi** [ether/ep_inline_toolbar#11]
We wanted to deinstall the inline toolbar again as it is hiding all menu icons in the topbar, even those not available in the toolbar. But the deinstallation hang up with error - we had to delete and restart manually.
![Screen Shot 2021-09-30 at 11 34 30](https://user-images.githubusercontent.com/1982011/135388361-27e8802c-b4af-438f-aa51-9d7a271d2a12.png)

**Comment by JohnMcLear** [comment]
Do you have any other plugin installed via git?

**Comment by lisandi** [comment]
No, only the etherpad itself gets installed via GIT and all other stuff via the plugin manager. Right now I try to figure out what plugins actually work as the plugin repo is a mass and many things not working or outdated or already integrated. This process will take probably still some days. Next step would be to set up an installation where all plugins work and against then all others could be tested. and for people interested then we have to set up a separate page which lists only the working plugins - which we can support in our sites which we set up - as otherwise it is getting a maintenance headache for us here. During that next step after testing each plugin we think also to check which functions actually could get gathered together at one place. and also we will look on how the plugins follow the same schemes as we found some i.e. workspaces - which have the templates at another place - so that we might try to correct than too.

You can only market a great product if you know what it can do and know how to maintain and to i.e. style it - there are a couple of plugins which would be much better with an icon i.e. all those dropdowns. The people using will be later children and students and people with nearly no it background so that simply has to be intuitive and simply working. I am sure we get there step by step as Etherpad is great! Especially with those workspaces - we only need to get those also working on a subfolder path - - many still have absolute links instead of relative links. 

Concerning deinstallation i woudl suggest to implement a "blocker" so that if one plugin gets deinstalled no other activity (installation or deinstallation" takes place - I guess that this might be a problem when some plugins install to slow and another action already starts.

**Comment by JohnMcLear** [comment]
1. Plugins from the Etherpad Foundation **mostly** have automated testing, mostly backend but some frontend.  For more info see https://github.com/ether/etherpad-lite/actions for core and the actions tab on each repo for each plugin.  We run thousands of tests each day to try and keep things stable but obviously it's easy for things to slip through the cracks and because we allow plugin authors autonomy that's a necessary evil (in my books).
2. RE Plugin polish / UX, absolutely, and thanks so much for beginning to fix these things up.  We do the best we can given that most of us also have full time professional/family roles and we mostly do Etherpad in our spare time so any contributions to the greater community is very well received ❤️.
3. I think we have pretty good test coverage for plugin install/deinstall but if you can tell me step by step how to replicate the issue I can add more test coverage and see if we can replicate in automated testing which means it lights a huge fire under the core teams ass to fix.  The trick here is always to replicate the problem from a clean install and document every tiny step you took.  Also it's best to do steps via the CLI because it saves us time during replication as clicks in are expensive vs the speed we can type.
